### ISSUE TITLE
Crush with editing label

### ISSUE THREAD

**Issue by Mollylulu** [wkentaro/labelme#1004]
when I try to modify the label to a new label, which does not exist in the list. It just report error and shut down the app.
<img width="411" alt="image" src="https://user-images.githubusercontent.com/26273127/158548514-5c0f33ed-31bf-4fd1-94a8-df749dcc8c7c.png">

labelme version == 5.0.1
conda env

**Comment by shihaoyin** [comment]
same problem

**Comment by syddpy666** [comment]
+1

**Comment by Roland-Pfeiffer** [comment]
same issue.
The problem seems to be _get_rgb_by_label() does not expect to receive an empty list, but UniqueLabelQListWidget().findItemsByLabel() returns an empty list if a new category is provided (as the case when an existing label gets updated with a completely new name).

```Traceback (most recent call last):
  File "<path-to-venv>/lib/python3.8/site-packages/labelme/app.py", line 1075, in editLabel
    self._update_shape_color(shape)
  File "<path-to-venv>/lib/python3.8/site-packages/labelme/app.py", line 1155, in _update_shape_color
    r, g, b = self._get_rgb_by_label(shape.label)
  File "<path-to-venv>/lib/python3.8/site-packages/labelme/app.py", line 1165, in _get_rgb_by_label
    item = self.uniqLabelList.findItemsByLabel(label)[0]
IndexError: list index out of range
```

**Comment by wkentaro** [comment]
Will be fixed by v5.0.2

```
pip install --no-cache-dir labelme==5.0.2
```
